docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+
+Over the years, the community grew to a point where we needed to examine its long-term needs and develop a system that will support the community without burning out the people who are filling key global roles.
+
+Last year we decided to try setting up a formalized a small, part-time paid staff with the goal of consolidating various roles and responsibilities that could be handled globally, such as web administration, swag coordination, staff wrangling, and meetup overseeing.
+
+So far the experiment has been successful, and the core team now includes the following dedicated documentarians:
+
+- Eric Holscher: Co-founder, head of North America conferences, global sponsorship coordinator
+- Mikey Ariel: Head of Europe conferences, global swag coordinator, communication chair
+- Samuel Wright: Head of support, Webmaster/GitHubmaster, meetup overseer, speaker coordinator (Europe)
+- Kelly O'Brien: Head of newsletter, job board coordinator, speaker coordinator (North America)
+
+These roles are on an including-but-not-limited-to basis, since there are lots of smaller, ad-hoc tasks that the core team runs into throughout the year.
+
+If you want to learn more about the organization behind the community, you can read :doc:`About Us </about-us>`. You can also find a high-level overview of the teams and tasks in :doc:`Team </team>`.
+
+Job Board
+---------
+
+As we mentioned in previous newsletters and Slack posts, we are launching a shiny new job board! Led by Kelly O'Brien, the job board aims to make it easy for documentarians to find their dream gigs, and for employers to find their dream documentarians.
+
+We're currently in the beta-testing phase, so if you're interested in helping us test the board – either as an employer or an applicant – we'd love to hear from you at `jobs@writethedocs.org <mailto:jobs@writethedocs.org>`_.
+
+Note: In order to help optimize our job-related efforts, we looked at our current communication channels, and came to the conclusion that the #jobs Slack channel together with the job board is the most likely to be a successful combination. Therefore, we are sadly closing the Write the Docs forum on (date?).
+
+Swag
+----
+
+Some of you might have noticed that this year's conference tickets do not include shirts. We realized that after 5 years of printing shirts for multiple conferences, the swag offering could use a bit of a refresh. In addition, the overhead of printing hundreds of shirts and managing inventory simply proved ineffective over time.
+
+So this year we are changing swag tactics, and instead of including shirts with the ticket purchase, we happily present **tote bags**! No sizing required, smoother production workflow, and just as collectible :)
+
+But have no fear, you can still get a shirt for your favorite conference! Inspired by the likes of DjangoCon US, we will team up with `TeeSpring <http://teespring.com/>`_, a trusted print-on-demand provider that works on a campaign-based model.
```

Comment:
Not sure if there's a reason to pre-announce a vendor here. Can probably just be vague and say you'll be able to order them online.